### ISSUE TITLE
Support for `alias` command?

### ISSUE THREAD

**Issue by HerbCSO** [asdf-vm/asdf#499]
### Steps to reproduce

I've just started the process of switching to `asdf` from `rbenv`/`jenv`/`rustup` and co. - mostly because shell startup times with loading all these version managers was getting kinda ridiculously slow and `asdf` is nice and fast - yay! The information on how to enable legacy version files (setting `legacy_version_file = yes` in `~/.asdfrc` was very helpful, and I was glad to see support for that (although it would be nice if it could be made more prominent in the ["Getting Started"](https://asdf-vm.github.io/asdf/#/core-manage-asdf-vm) section for newcomers like myself).

However, I'm running into a slight niggle with the legacy version support in that the versions that things like `jenv` provide don't map neatly to what for example the `java` plugin supports. E.g. `jenv` knows about a `1.8` alias, which I currently have mapped to `1.8.0.181`, and that in turn points to the actual version of `oracle64-1.8.0.181` (yeah, I know, I'm a little behind ;] ). So my `.java-versions` file points to `1.8` and that all works fine under `jenv`. Now I'd like to make that work in the `asdf java` plugin, but I don't really see a good mechanism to do so (aside from the fact that I needed to [add a `list-legacy-filenames` to the plugin](https://github.com/skotchpine/asdf-java/pull/56/files), but that's beside the point here).

I've thought about adding a `parse-legacy-file` command to the `java` plugin, and that works - a working (not particularly good) example for this particular case would be:

```
#!/usr/bin/env bash
parse_legacy_version() {
  local version=$(cat $1)
  case $version in
    1\.8)
      echo "oracle-8.202"
      ;;
    *)
  esac
}

parse_legacy_version $1
```

However this seems a bit convoluted for what seems to be a simple use case.

A better way that also _seems_ to work is to add the alias manually by running something like `ln -s ~/.asdf/installs/java/oracle-8.202 ~/.asdf/installs/java/1.8` (yeah, I went and upgraded to a newer version while I was at it ;] ). This is reasonable but slightly annoying for what seems like it would be a common use case. Also, I'm not 100% certain yet that this would work for all use cases.

### Expected behavior

There should be support for an `alias` command of some sort like `jenv` does automatically or like the [`rbenv-aliases`](https://github.com/tpope/rbenv-aliases) plugin.

### Actual behavior

Need to manually create the alias (yeah, I know, woe is me, this is kinda whiny ;] ).

### Environment

**OS**: macOS Sierra 10.12.6

**asdf version**: 0.7.0

**Comment by Stratus3D** [comment]
I think this is related to #352, though it's not an exact duplicate.

**Comment by HerbCSO** [comment]
Thanks so much for the quick response!

Yes, #352 does look very much like it, and after reading the very considered discussion there I very much agree with the points made and I'm happy to close this issue in favor of #352. Sorry, I only searched specifically for `alias` before I submitted this and didn't find that issue as a result.

PS: Thanks also for including the reference in the core setup guide! I've fixed a little typo in #501.
